pandapower/plotting/plotly/traces.py

```diff
@@ -920,43 +935,43 @@ def create_scale_trace(net, weighted_trace, down_shift=0):
     """
     marker = weighted_trace["marker"]
     scale_info = weighted_trace["meta"]
+    unit = scale_info["scale_legend_unit"]  # p_mw...q_mvar ..
+
     # scale trace
     x_max, y_max = net.bus_geodata.x.max(), net.bus_geodata.y.max()
     x_min, y_min = net.bus_geodata.x.min(), net.bus_geodata.y.min()
-
     x_pos = x_max + (x_max - x_min) * 0.2
     x_pos2 = x_max + ((x_max - x_min) * 0.2 * 2)
-    y_pos = y_max - ((y_max - y_min) * (0.2 * down_shift))
-
-    # p_mw...q_mvar ..
-    unit = scale_info["column_to_plot"].split("_")[1].upper()
 
-    # default is the average rounded to 5
-    if not scale_info["scale_marker_size"]:
-        scale_size = math.ceil(marker["size"].mean() / 5) * 5
+    scale_traces = []
+    for idx, sze in enumerate(scale_info["scale_marker_size"]):
+        y_pos = y_max - ((y_max - y_min) * (0.2 * (down_shift + idx)))
 
-    else:
-        scale_size = scale_info["scale_marker_size"] * scale_info['marker_scaling']
-
-    # second (dummy) position is needed for correct marker sizing
-    scale_trace = dict(type="scatter",
-                        x=[x_pos, x_pos2],
-                        y=[y_pos, y_pos],
-                        mode="markers+text",
-                        hoverinfo="skip",
-                        marker=dict(size=[scale_size, 0],
-                                    color=marker['color'],
-                                    symbol=marker["symbol"],
-                                    sizemode=marker["sizemode"]),
-                        text=[f"scale: {scale_size / scale_info['marker_scaling']} {unit}", ""],
-                        textposition="top center",
-                        showlegend=False,
-                        textfont=dict(
-                                        family="Helvetica",
-                                        size=14,
-                                        color="DarkSlateGrey"))
-
-    return scale_trace
+        # default is the average rounded to 5
+        if not sze:
+            scale_size = math.ceil(marker["size"].mean() / 5) * 5
+        else:
+            scale_size = sze * scale_info['marker_scaling']
+
+        # second (dummy) position is needed for correct marker sizing
+        scale_trace = dict(type="scatter",
+                           x=[x_pos, x_pos2],
+                           y=[y_pos, y_pos],
+                           mode="markers+text",
+                           hoverinfo="skip",
+                           marker=dict(size=[scale_size, 0],
+                                       color=scale_info.get("scale_marker_color"),
+                                       symbol=marker["symbol"],
+                                       sizemode=marker["sizemode"]),
+                           text=[f"scale: {scale_size / scale_info['marker_scaling']} {unit}", ""],
```

Comment:
I'd like to make the `"scale:" ` part more variable, e.g. for plots in other languages than English. We could add something like `scale_marker_text_prefix` to `weightedmarkertrace["meta"]` and add `scale_marker_text_prefix='scale: '` to the input arguments of `create_weighted_marker_trace`.
What do you think about that, @jwiemer112 ?

Reply:
As bilateral discussed, I removed the text from the scale legend. :)